EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||ads-partners.coupang.com^$third-party
```

Comment:
Move it to the `adservers_firstparty.txt` with `$third-party`, if it's necessary.

Reply:
This domain is globally used by many website.
I think Moving to `adservers_firstparty.txt` is unnecessary.

Reply:
Well, I don't see a reason that the domain has to be in `adservers.txt`. It's subdomain.

Reply:
https://partners.coupang.com/ ([Google Translate](https://translate.google.com/translate?hl=&sl=auto&tl=en&u=https%3A%2F%2Fpartners.coupang.com%2F))
https://partners.coupang.com/#help/tag-usage-guide ([Google Translate](https://translate.google.com/translate?sl=auto&tl=en&u=https://partners.coupang.com/%23help/tag-usage-guide))

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/90552/commits/ca58117cb25c5b3684b4d8d728c18be9260eaa38

Reply:
Ok, but why not `adservers_firstparty.txt`? Subdomains should be there and we usually add them without `$third-party`.

Reply:
Oh, Okay. I'll move it.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/90552/commits/76f049ecec88b6a094ff74472d9b3d6621b2ac7a